### PR TITLE
Updated Readme Document of jest testing

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
Updated Readme Document of jest testing

### CODE REVIEW

**Comment by AmbrishRamachandiran** [comment]
Awaiting review commenting for avoid stale

**Comment by AmbrishRamachandiran** [comment]
@Rugvip please review ! commenting for avoid stale

**Comment by AmbrishRamachandiran** [comment]
please review ! commenting for avoid stale

File: docs/plugins/testing.md [hunk 1]
```diff
@@ -26,10 +26,14 @@ To run both `MyComponent.test.tsx` and `MyControl.test.tsx` suite of tests:
 
     yarn test MyCo
 
-Note: if `console.logs` are not appearing, run only the individual test you are
+:::note Note
+
+if `console.logs` are not appearing, run only the individual test you are
```
Comment (Rugvip) [inline docs/plugins/testing.md:31]: While at it, we could update this to `console.log`s

**Review by Rugvip — COMMENTED** [review]
Thank you!

Minor bonus thing

**Review by Rugvip — APPROVED** [review]
Thanks! 👍

**Pull request merged**